README.md

```diff
@@ -0,0 +1,46 @@
+# Typescript Wallet SDK Monorepo
+
+Typescript Wallet SDK is a library that allows developers to build wallet
+applications on the Stellar network faster. To jump right into how the main
+typescript-wallet-sdk module works you can
+[go here](./@stellar/typescript-wallet-sdk/README.md).
+
+## Yarn Workspaces
+
+This repo uses a yarn worspace to create a monorepo that includes the main
+wallet sdk, and two modules for extending functionality:
+
+- [typescript-wallet-sdk](./@stellar/typescript-wallet-sdk/README.md) - the main
```

Comment:
Probably a good idea to add yarn install command for all listed sub projects

Reply:
good call
https://github.com/stellar/typescript-wallet-sdk/pull/136/commits/bbd2f6cc4fbb7635a2e52e268e7229eeae46b5a7